src/js/views/patients/schedule/schedule_views.js

```diff
@@ -149,34 +149,45 @@ const DayItemView = View.extend({
     'click .js-patient-sidebar-button': 'click:patientSidebarButton',
     'click .js-patient': 'click:patient',
     'click': 'click',
-    'click .js-select': 'click:select',
+  },
+  modelEvents: {
+    'change': 'render',
```

Comment:
So we needed this on the other list to handle bulk editing.  But this app doesn't seem to depend on that.. might need to implement the other lists however schedule is working, but I don't think this event is needed

Reply:
Without this event, the list items don't update in the UI after saving a bulk edit change (update status, date, etc.).

On successful bulk select, `select:none` is fired off. Which the schedule view listens for and calls`this.render()`. But now that doesn't happen since we call `this.showCheck()` instead of `this.render()`.

I believe the schedule and worklist/patient flow lists work the same in this regard.